test/e2e/upgrade_test.go

```diff
@@ -0,0 +1,61 @@
+// +build upgrade
+
+package e2e
+
+import (
+	"context"
+	"github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"os"
+	"strings"
+	"testing"
+)
+
+const EnvUpdateVersionKey = "UPDATE_TEST_VERSION"
+const upgradeTestTag = "next"
+
+func TestOperatorUpgrade(t *testing.T) {
+
+	upgradeTestVersion := os.Getenv(EnvUpdateVersionKey)
+	t.Log(upgradeTestVersion)
```

Comment:
Add some context to this log entry. Like, `Attempting to upgrade to version ...`

Reply:
This log should not be here, was part of my debugging, but yeah a log message could be useful.